js/app.js

```diff
@@ -0,0 +1,163 @@
+const a = 4;
+const b = 53;
+const c = 57;
+const d = 16;
+const e = 'Kevin';
+
+console.log(a < b);
+console.log(c > d);
+console.log('Name' === 'Name');
+console.log(a < b < c);
+console.log(a <= a <= d);
```

Comment:
clever, but it doesn't evaluate to "true" for teh reasons you might think.

`a * a === d`